plugins/modules/mount.py

```diff
@@ -247,6 +247,10 @@ def _set_mount_save_old(module, args):
             '%(src)s - %(name)s %(fstype)s %(passno)s %(boot)s %(opts)s\n')
 
     for line in open(args['fstab'], 'r').readlines():
+        # Append newline if the line in fstab does not finished with newline.
```

Comment:
@saito-hideki Thanks for the PR. Would it possible to add a test for this change? Thanks.

Reply:
@Akasurde sure thing! I'll add integration tests

Reply:
done! :)